naptan_update/update_table.py

```diff
@@ -47,23 +46,21 @@ def lambda_handler():
         )
     except ClientError as e:
         print(e.response['Error']['Message'])
-        raise
+        raise CannotGetModificationDate(e.response['Error']['Message'])
```

Comment:
In such a case, it probably means the item isn't there and we should print a meaningful message that create should be called at least once before - or something similar